src/app/lib/utilities/srcSet/placeholder.js

```diff
@@ -1,7 +1,7 @@
-const getPlaceholderSrc = src => {
+const getPlaceholderSrc = (src, imageWidth = '512') => {
```

Comment:
Does this default need to be specified both here and at https://github.com/bbc/simorgh/pull/4362/files#diff-3a458af3a03c69f75416a5c352175b83R80?